nservicebus/azure-service-bus/configuration/full.md

```diff
@@ -110,6 +110,7 @@ The following settings determine how NServiceBus will connect to Azure Service B
 
  * `NumberOfClientsPerEntity(int)`: NServiceBus maintains a pool of receive and send clients for each entity. This setting determines how big that pool is. Defaults to 5.
  * `ConnectivityMode(ConnectivityMode)`: Determines how NServiceBus connects to Azure Service Bus, using TCP or HTTP. Defaults to TCP.
+ * `TransportType(TransportType)`: Determines what transport protocol NServiceBus is using for Azure Service Bus, `TransportType.NetMessaging` or `TransportType.Amqp`. Defaults to `TransportType.NetMessaging`.
```

Comment:
@yvesgoeleven - How about this?

>* `TransportType(TransportType)`: Determines what transport protocol NServiceBus is using for Azure Service Bus, `TransportType.NetMessaging` or `TransportType.Amqp`. Defaults to `TransportType.NetMessaging`.

* `TransportType(TransportType)`:  This setting determines what transport protocol NServiceBus is using for Azure Service Bus. The available options are: `TransportType.NetMessaging` and `TransportType.Amqp`. The defaults option is set to `TransportType.NetMessaging`.